notebooks/workflows/amun-inspection-workflow.ipynb

```diff
@@ -10,7 +10,7 @@
    },
```

Comment:
<p>Same comment as above regarding the inspection id</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/thoth-station/notebooks/pull/55/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>